### PR TITLE
166 Update DG

### PR DESCRIPTION
**Pull request by NgChunMan**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -90,29 +90,29 @@ Here's a (partial) class diagram of the `Logic` component:
 
 <puml src="diagrams/LogicClassDiagram.puml" width="550"/>
 
-The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("delete 1")` API call as an example.
+The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("rmt 1")` API call as an example.
```
Comment (mingyuanc) [inline docs/DeveloperGuide.md:93]: maybe can add what rmt 1 means

**Review by mingyuanc — CHANGES_REQUESTED** [review]
Fast change

**Review by NereusWB922 — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (mingyuanc) [reply docs/DeveloperGuide.md:93]: you did not add what rmt 1 means

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -90,29 +90,30 @@ Here's a (partial) class diagram of the `Logic` component:
```
Comment (mingyuanc) [inline docs/DeveloperGuide.md:90]: image
Change the link to our repo one

**Review by mingyuanc — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (NgChunMan) [reply docs/DeveloperGuide.md:103]: Good catch Ming Yuan. The link is now the updated version.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (NgChunMan) [reply docs/DeveloperGuide.md:106]: Is line 94 the description for command 'rmt 1' not clear?

**Review by mingyuanc — APPROVED** [review]
Lgtm

**Pull request merged**